lib/protocol/networks.js

```diff
@@ -627,6 +653,13 @@ testnet.block = {
   slowHeight: 0
 };
 
+testnet.airdrop = {
+  disableGoosig: true,
+  // TODO: see what this should be relative
+  // to the rest of the network.
+  disableGoosigHeight: (365 - 30) * testnet.pow.blocksPerDay
```

Comment:
Just taking a look at main `auctionstart=10` vs testnet `auctionstart=.25` the correlation is main*.025 = testnet. 

So I think if we apply the same logic here (365+30)*.025 = ~10 days. That seems a bit too early, so I'd suggest maybe 20 days for this on testnet?

Reply:
I agree that it does feel early, 20 days sounds good to me. If there was a really easy way for people to do their claims in regtest, that would be most ideal.